app/wrappers/prospect_api_wrapper.rb

```diff
@@ -0,0 +1,45 @@
+class ProspectApiWrapper
+  require 'rest-client'
+
+  API_KEY = 'a2d2142ae77ee3ff86fd6ac644e339d9'.freeze
+  BASE_URL = 'http://prospects-api.herokuapp.com/'.freeze
+  SUCCESS = 200
+
+  def self.get_prospects
+    response = get('prospects')
+
+    if success?(response)
+      prospects = JSON.parse(response.body).with_indifferent_access
```

Comment:
it seems that `symbolize_keys` works only for a first level of keys...  `get` will return properly formatted response 👍